casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -635,4 +635,24 @@ object BlockAPI {
       )
     )
   }
+
+  def bondStatus[F[_]: Monad: EngineCell: Log](
+      publicKey: ByteString
```

Comment:
Is this a private key, public key, validator identity, etc? I believe we have specific types for each one.